File: 400/453 - Minimum Moves to Equal Array Elements.c
#include <limits.h>

int minMoves(int* nums, int numsSize) {
    int sum = 0, i, min = INT_MAX;

    for (i = 0; i < numsSize; ++i) {
        sum += nums[i];
        if (nums[i] < min)
            min = nums[i];
    }
    
    return sum - min * numsSize;
}

File: csharp/453.minimum-moves-to-equal-array-elements.cs
/*
 * @lc app=leetcode id=453 lang=csharp
 *
 * [453] Minimum Moves to Equal Array Elements
 */
public class Solution {
    public int MinMoves(int[] nums) {
        long sum = 0, min = int.MaxValue;
        foreach (int num in nums) {
            min = Math.Min(min, (long) num);
            sum += (long) num;
        }
        return (int) (sum - min * (long) nums.Length);
    }
}

